notebooks/experimental/prediction_tensorflow_model.ipynb

```diff
@@ -0,0 +1,1211 @@
+      "Requirement already satisfied: pyasn1-modules>=0.2.1 in /home/ksenia-evin/.local/lib/python3.8/site-packages (from google-auth<2,>=1.6.3->tensorboard~=2.6->tensorflow==2.6.0) (0.2.8)\n",
+      "Requirement already satisfied: rsa<5,>=3.1.4; python_version >= \"3.6\" in /home/ksenia-evin/.local/lib/python3.8/site-packages (from google-auth<2,>=1.6.3->tensorboard~=2.6->tensorflow==2.6.0) (4.7.2)\n",
+      "Requirement already satisfied: cachetools<5.0,>=2.0.0 in /home/ksenia-evin/.local/lib/python3.8/site-packages (from google-auth<2,>=1.6.3->tensorboard~=2.6->tensorflow==2.6.0) (4.2.2)\n",
+      "Requirement already satisfied: oauthlib>=3.0.0 in /usr/lib/python3/dist-packages (from requests-oauthlib>=0.7.0->google-auth-oauthlib<0.5,>=0.4.1->tensorboard~=2.6->tensorflow==2.6.0) (3.1.0)\n",
+      "Requirement already satisfied: pyasn1<0.5.0,>=0.4.6 in /home/ksenia-evin/.local/lib/python3.8/site-packages (from pyasn1-modules>=0.2.1->google-auth<2,>=1.6.3->tensorboard~=2.6->tensorflow==2.6.0) (0.4.8)\n"
+     ]
+    }
+   ],
+   "source": [
+    "!pip install pandas==1.3.2 numpy==1.19.5 sklearn==0.0 geopy==2.2.0 scipy==1.7.1 seaborn==0.11.2 matplotlib==3.4.3 tensorflow==2.6.0"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "a3f5160c-4f05-4dc1-9e6a-d044dd81947f",
+   "metadata": {},
+   "source": [
+    "## Imports"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "id": "b45c477b-a2e3-4783-b4d7-6771126de879",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "2021-09-15 15:22:57.246682: W tensorflow/stream_executor/platform/default/dso_loader.cc:64] Could not load dynamic library 'libcudart.so.11.0'; dlerror: libcudart.so.11.0: cannot open shared object file: No such file or directory\n",
+      "2021-09-15 15:22:57.246705: I tensorflow/stream_executor/cuda/cudart_stub.cc:29] Ignore above cudart dlerror if you do not have a GPU set up on your machine.\n"
+     ]
+    }
+   ],
+   "source": [
+    "from uuid import uuid4\n",
+    "import requests\n",
+    "\n",
+    "import matplotlib.pyplot as plt\n",
+    "import numpy as np\n",
+    "import pandas as pd\n",
+    "import seaborn as sns\n",
+    "import tensorflow as tf\n",
+    "from geopy.distance import geodesic\n",
+    "from scipy.cluster.vq import vq\n",
+    "from sklearn.cluster import KMeans\n",
+    "from sklearn.model_selection import train_test_split\n",
+    "from sklearn.preprocessing import StandardScaler\n",
+    "\n",
+    "from tensorflow import keras\n",
+    "from tensorflow.keras import Sequential, optimizers\n",
+    "from tensorflow.keras.layers import Dense, Flatten, Softmax\n",
+    "\n",
+    "from unfolded.map_sdk import UnfoldedMap"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "6fd123df-3aa2-4610-b02e-b555c152cf5e",
+   "metadata": {},
+   "source": [
+    "Check package versions: "
```

Comment:
Do we really need this section?

Reply:
I guess, yes. Because if, for example, you already have all the required package in your env, you don't need to run the first installation cell. For this reason, we need to check your versions of packages